tests/test-base.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env batsPI
+
+load globals
+load test_helper
+
+
+@test "CKAN container is up" {
+  wait_for_app
+}
+
+@test "/user/login is up" {
+  test_view_login
```

Comment:
Are these functions reusable? Why not just incline the code so we can see the test description with the test code?

Reply:
I transformed this function to be reusable [here](https://github.com/GSA/catalog.data.gov/pull/10/commits/4ef22440d2e556bc67ef2f477467e8c2dda8db30)